Allow attempts to print PDF and PS files using the LPR backend.
https://gitlab.gnome.org/GNOME/gtk/-/commit/8d5357ee56b1d34fe14346ed15004f9e4d571594

Index: modules/printbackends/lpr/gtkprintbackendlpr.c
--- modules/printbackends/lpr/gtkprintbackendlpr.c.orig
+++ modules/printbackends/lpr/gtkprintbackendlpr.c
@@ -392,9 +392,13 @@ gtk_print_backend_lpr_init (GtkPrintBackendLpr *backen
 {
   GtkPrinter *printer;
 
-  printer = gtk_printer_new (_("Print to LPR"),
-			     GTK_PRINT_BACKEND (backend),
-			     TRUE); 
+  printer = g_object_new (GTK_TYPE_PRINTER,
+			  "name", _("Print to LPR"),
+			  "backend", backend,
+			  "is-virtual", FALSE,
+			  "accepts-pdf", TRUE,
+			  "accepts-ps", TRUE,
+			  NULL);
   gtk_printer_set_has_details (printer, TRUE);
   gtk_printer_set_icon_name (printer, "gtk-print");
   gtk_printer_set_is_active (printer, TRUE);
